From 73b78347959e579ed2a1f008faa8a159d9bd9d1c Mon Sep 17 00:00:00 2001
From: Janne Grunau <j@jannau.net>
Date: Sat, 20 Apr 2024 08:59:29 +0200
Subject: [PATCH 1/1] FEDORA: configs: apple: Disable AUTOBOOT_KEYED

Fedora patches this to 'default y' but the behavior on
Fedora-Asahi-Remix was to interrupt boot with any key. Keep this
behavior.

Signed-off-by: Janne Grunau <j@jannau.net>
---
 configs/apple_m1_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/apple_m1_defconfig b/configs/apple_m1_defconfig
index 54608bf123f..057313938e1 100644
--- a/configs/apple_m1_defconfig
+++ b/configs/apple_m1_defconfig
@@ -4,6 +4,7 @@ CONFIG_ARCH_APPLE=y
 CONFIG_DEFAULT_DEVICE_TREE="apple/t8103-j274"
 CONFIG_SYS_BOOTM_LEN=0x800000
 CONFIG_SYS_LOAD_ADDR=0x0
+# CONFIG_AUTOBOOT_KEYED is not set
 # CONFIG_AUTOBOOT_MENU_SHOW is not set
 CONFIG_BOOTCOMMAND="bootflow scan -b"
 CONFIG_USE_PREBOOT=y
